fares-data-build-tool/create-data
# feat: CFD-299 View Purchase Methods

# Description

-   View purchase methods in global setting

# Testing instructions

-   Check you can view previously created SOPs in global settings
- NOTE: edit/delete/create will not work yet.

# Type of change
